src/routes/costModels/costModelsDetails/bottomPagination.tsx

```diff
@@ -7,6 +7,8 @@ import {
   ToolbarItem,
   ToolbarItemVariant,
 } from '@patternfly/react-core';
+import { intl } from 'components/i18n';
```

Comment:
After looking at the code, again. We can easily change this function to be a function component, then we could use injectIntl.

```
type BottomPaginationBaseProps = Omit<PaginationProps, 'ref'> & WrappedComponentProps;

const BottomPaginationBase: React.FC<BottomPaginationBaseProps> = props => {
```

Reply:
We can also go a step further and refactor it to look like this.
```
type BottomPaginationBaseProps = Omit<PaginationProps, 'ref'> & WrappedComponentProps;

const BottomPaginationBase: React.FC<BottomPaginationBaseProps> = ({
  itemCount,
  intl,
  onPerPageSelect,
  onSetPage,
  page,
  perPage,
  variant,
}) => (
  <Toolbar>
    <ToolbarContent>
      <ToolbarItem variant={ToolbarItemVariant.pagination}>
        <Pagination
          onPerPageSelect={onPerPageSelect}
          onSetPage={onSetPage}
          variant={variant}
          itemCount={itemCount}
          page={page}
          perPage={perPage}
          titles={{
              paginationTitle: intl.formatMessage(messages.paginationTitle, {
                  title: intl.formatMessage(messages.costModelsDetailsTitle),
                  placement: 'bottom',
              }),
          }}
        />
      </ToolbarItem>
    </ToolbarContent>
  </Toolbar>
);
```